### PR TITLE
Visible indicators

### PR DESCRIPTION
**Pull request by Reettaphant**
Mobile and Desktop visibility indicators on collection items! Jon already told me that there is no way I'm getting `properStages` as a variable name through peer review so I welcome any better suggestions :).
screen shot 2018-11-16 at 12 22 39

### CODE REVIEW

File: client-v2/src/shared/types/Collection.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ interface Group {
   articleFragments: string[];
 }
 
+type ProperStages = 'draft' | 'live';
```
Comment (jonathonherbert) [inline client-v2/src/shared/types/Collection.ts]: Suggestions from the team - CurrentStages? ActiveStages?

Or, we could name ProperStages as Stages and rename Stages to CollectionItemSets or similar?

File: client-v2/src/actions/Collections.ts [hunk 2]
```diff
@@ -3,77 +3,97 @@ import { batchActions } from 'redux-batched-actions';
...
-          batchActions([
-            collectionActions.fetchSuccess(collection),
-            articleFragmentsReceived(articleFragments),
-            recordUnpublishedChanges(collectionId, hasUnpublishedChanges),
-            groupsReceived(groups)
-          ])
-        );
+      dispatch(
+        batchActions([
+          recordVisibleStories(collection.id, liveVisibleStories, frontStages.live),
```
Comment (RichieAHB) [inline client-v2/src/actions/Collections.ts]: Is there a way we can create this action with a shape that means we only have to call it once with a subset of either `live` or `draft` in the payload? Maybe that isn't desirable anyway?

File: client-v2/src/util/frontsUtils.ts [hunk 3]
```diff
@@ -43,9 +47,26 @@ const isFrontStale = (lastUpdated?: number, lastPressed?: number) => {
   return false;
 };
 
+const getVisibilityStoryDetails = (groupsWithStories: ArticleFragment[][]) => groupsWithStories.reduce((stories, storiesInGroup, index) => {
```
Comment (RichieAHB) [inline client-v2/src/util/frontsUtils.ts]: I think I need to understand stories a bit more - not quite sure what this is doing ...

Comment (Reettaphant) [reply client-v2/src/util/frontsUtils.ts]: Fear not, stories are just badly named articles. This has been fixed now.

**Pull request merged**